app/views/options.html

```diff
@@ -31,9 +31,87 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+<div>
+	<button id="login" style="display:none">Log in with Dropbox</button>
+	<a id="logout" href="#" style="display:none">Log out</a>
+	<button id="dbsave" style="display:none">save to Dropbox</button>
+	<button id="dbreset" style="display:none">reset Dropbox</button>
+	<script>
```

Comment:
yes..ok.. i've never used angularjs or coffeescript  before..  i've read up a bit overnight.. so i'll try and fix that for next revision..